app/views/application/previewapplication/schemePreferencesViewSection.scala.html

```diff
@@ -19,7 +19,7 @@ <h2 class="heading-large">
             @for(position <- 0 to selectedSchemes.schemes.size - 1) {
```

Comment:
matter of preference. I would have preferred use of 'until' rather than 'to' here.